internetarchive/openlibrary
# Added yearly goal affordances to mobile view in my books page

Closes #10168

Fix

At present the UI does not have the option to set the yearly goals in Mobile view (as it is present in the side bar which is not included in the mobile view). This PR adds the option to set the yearly goals in the main body in mobile view.

### Testing

1. Deploy the branch locally.
2. Go to my books page.
3. Look on the "set yearly goals" section.
4. Observe the responsiveness in mobile and laptop view. 

### Screenshot

![WhatsApp Image 2024-12-27 at 5 52 04 PM](https://github.com/user-attachments/assets/b26944a6-6b39-4d20-b989-572fb982c4da)

**After setting a particular goal**
![WhatsApp Image 2024-12-27 at 5 52 04 PM (1)](https://github.com/user-attachments/assets/a03b5286-0541-4d2e-8e2b-b7bff25d4af3)

**In laptop view**
![WhatsApp Image 2024-12-27 at 5 53 18 PM](https://github.com/user-attachments/assets/64af84f6-23ef-4b49-bf52-fca426d58355)


### Stakeholders

@jimchamp @mekarpeles
